theme/enums.py

```diff
@@ -5,6 +5,5 @@ class QuotaStatus(str, enum.Enum):
     """Quota status message type"""
 
     ENFORCEMENT = 'enforcement'
-    GRACE_PERIOD = 'grace_period'
```

Comment:
I still find some lingering `remaining_grace_period` that need to be removed.
Example:
https://github.com/hydroshare/hydroshare/blob/5680-minio-quota/theme/utils.py#L56